### ISSUE TITLE
[Feat] Allow multiple maps to be turned on/off in Display tab simultaneously

### ISSUE THREAD

**Issue by ericrpatton** [OSGeo/grass#1277]
**Is your feature request related to a problem? Please describe.**
I frequently work with many maps loaded in the Display tab, and if I want to turn on/off the visibility of a subset of them in the Map Display, I need to click each one individually, which can be frustrating.

**Describe the solution you'd like**
I'd like the ability to right-click on a set of selected maps in the Display tab, and be able to 'Display all' if none of them are currently turned on and visible in the Map Display, or 'Un-Display all' if they are all visible. 

**Describe alternatives you've considered**
The only alternative is to turn on or off each map individually, which can be painful.

**Additional context**
Currently the only option in the right-click context menu (when multiple maps are selected) is Remove.

**Comment by ericrpatton** [comment]
I can't seem to find a way to add a label to a feature request after the ticket is created; this should be labelled as a GUI request.

**Comment by petrasovaa** [comment]
An alternative is using the groups, each group can be checked/unchecked and that would apply to all the layers in the group. There is 'Add group' in the layer manager toolbar.

**Comment by ericrpatton** [comment]
Hi Anna. True, that is a workaround. To make that workaround easier, it would be great to be able to add multiple maps to the group in one drag-and-drop move. As it is now, each map has to be added individually.

**Comment by ericrpatton** [comment]
Nevermind, I just discovered that you can add multiple maps to a group through the CTRL-SHIFT-L add maps dialog - nice!

**Comment by mlennert** [comment]
> Nevermind, I just discovered that you can add multiple maps to a group through the CTRL-SHIFT-L add maps dialog - nice!

Does this mean your issue is actually solved and can be closed ?

**Comment by ericrpatton** [comment]
> Does this mean your issue is actually solved and can be closed ?

I would say no; it would still be very useful to be able to turn off multiple maps based on an arbitrary selection rather than group them first, which seems like a counter-intuitive and unnecessary step. I use groups mainly as a way of organizing maps thematically. Also, grouping the maps makes all of the grouped members inherit the same drawing order in the Display tab, which may not be what the user wishes, either.

**Comment by petrasovaa** [comment]
I agree, it should be relatively simple to add "Check selected layers" and "Uncheck selected layers" into the context menu.
